corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
Ok I'll add it to the `BULK_CASE_MANAGEMENT`. One clarification though, you still want this under a toggle even after it's added as to a Role. Also @czue you could not have picked a better few weeks to escape from boston haha